addons/account/static/tests/test_payment_term_line_widget.js

```diff
@@ -0,0 +1,103 @@
+/** @odoo-module **/
+import { click, getFixture } from "@web/../tests/helpers/utils";
+import { makeView , setupViewRegistries } from "@web/../tests/views/helpers";
+
+let serverData;
+let target;
+
+QUnit.module('Widget', (hooks) => {
+    QUnit.module("PaymentTermsLineWidget");
+    hooks.beforeEach(() => {
+        target = getFixture();
+        serverData = {
+            models: {
+                account_payment_term: {
+                    fields: {
+                        line_ids: {
+                            string: "Payment Term Lines",
+                            type: "one2many",
+                            relation: "account_payment_term_line",
+                        },
+                    },
+                    records: [
+                        {
+                            id: 1,
+                            line_ids: [1, 2],
+                        },
+                    ],
+                },
+                account_payment_term_line: {
+                    fields: {
+                        value: {
+                            string: "Value",
+                            type: "selection",
+                            selection: [["balance", "Balance"], ["percentage", "Percentage"]],
+                        },
+                        value_amount: { string: "Due", type: "float" },
+                        delay_type: {
+                            string: "Delay Type",
+                            type: "selection",
+                            selection: [["days_after", "Days after invoice date"], ["days_after_end_of_month", "Days after end of month"], ["days_after_end_of_next_month", "Days after end of next month"]],
+                        },
+                        nb_days: { string: "Number of Days", type: "integer" },
```

Comment:
I think we can simplify the test models
we don't really need this many fields in the one2many comodel
The idea is to test the js behaviour with any list renderer, so keep it simple

Reply:
sure I willl minimize it !